src/main/java/gregtech/api/metatileentity/multiblock/RecipeMapMultiblockController.java

```diff
@@ -140,6 +146,19 @@ private void resetTileAbilities() {
         this.energyContainer = new EnergyContainerList(Lists.newArrayList());
     }
 
+    protected IMultipleTankHandler extendedImportFluidList(IMultipleTankHandler fluids) {
+        List<IFluidTank> tanks = new ArrayList<>(fluids.getFluidTanks());
+        for (var handler : getAbilities(MultiblockAbility.IMPORT_ITEMS)) {
```

Comment:
A quick comment here that we are using import Items to handle any Dual Handlers would be appreciated